dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/ApplicationInstallInfo.java

```diff
@@ -81,6 +84,12 @@ public boolean getUseJandex() {
         return _config.getUseJandex();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String getAnnotationScanLibrary() {
+        return _config.getAnnotationScanLibrary();
```

Comment:
```suggestion
        return (String) _config.getConfigProperty(AppManagerConstants.ANNOTATION_SCAN_LIBRARY);
```
Of course you will need to add the import for AppManagerConstants as well.